apps/web/screens/NewsItem.tsx

```diff
@@ -69,7 +87,16 @@ const NewsItem: Screen<NewsItemProps> = ({ newsItem }) => {
   return (
     <>
       <Head>
-        <title>{newsItem.title} | Ísland.is</title>
+        <title>{metaTitle}</title>
```

Comment:
I would create a `<Meta title={metaTilte} description={metaDescription} image={{ url: newsItem.image, width: newsItem.image.height.toString()  }} />` component for this where you can handle the default values in one place only and then pass your object with the custom title/image/dimensions.

Besides, you might also want to add twitter cards metadata to support share link there (cf: `twitter:title`, `twitter:card`, `twitter:image`, `twitter:site`, `twitter:description`, https://cards-dev.twitter.com/validator)

Reply:
Yep yep, good point! I'll create the component and handle twitter support as well